datanode/snapshot/config.go

```diff
@@ -0,0 +1,24 @@
+package snapshot
+
+import "code.vegaprotocol.io/vega/datanode/config/encoding"
+
+type Config struct {
+	RemoveSnapshotsOnStartup encoding.Bool `long:"remove-snapshops-on-startup" description:"if true the nodes snapshot directory will be emptied at startup"`
+
+	BlockInterval int64         `long:"block-interval" description:"the block interval between creation of snapshots, 0 if snapshot create should be disabled (default: 0)"`
+	Publish       encoding.Bool `long:"publish" description:"set to true if this node should publish snapshot data"`
+	PanicOnError  encoding.Bool `long:"panic-on-error" description:"true if service should panic on snapshot creation error"`
+
+	DatabaseSnapshotsPath string `long:"database-snapshot-path" description:"the snapshots path relative the database working directory"`
+}
+
+// NewDefaultConfig creates an instance of the package specific configuration, given a
+// pointer to a logger instance to be used for logging within the package.
+func NewDefaultConfig() Config {
+	return Config{
+		DatabaseSnapshotsPath:    "/snapshots",
```

Comment:
Shouldn't this be set in the paths library? cc @ValentinTrinque

Reply:
I think it should be in the paths library too, so when you do `vega paths list` you can see the default location for where things get dumped. It then opens up us having an absolute path here, which is a bit more flexible.

Reply:
related to later comment, the db snapshot path is a path relative to where the DB is run, different to the nodes snapshot path, the nodes snapshot path is part of the paths list.  The db path will depend on where and how the DB is run, its sets to the most common value in config, but we don't mandate that.

Reply:
yeah, they have to be on the same machine unless using some network disk etc